Controls/ReactBridge/src/DotVVM.Contrib/ChartData.cs

```diff
@@ -0,0 +1,20 @@
+﻿using DotVVM.Framework.ViewModel;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace DotVVM.Contrib
+{
+    public class ChartData
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IEnumerable<string> labels { get; set; }
+        [Bind(Direction.None)]
+        public IEnumerable<string> Labels { get => labels; set => labels = value; }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IList<IEnumerable<double>> series { get; set; }
+        [Bind(Direction.None)]
+        public IList<IEnumerable<double>> Series { get => series; set => series = value; }
```

Comment:
:) This is fancy trick, I'd just like to say, that as of dotvvm 1.2 it's possible to client-side rename property using Bind(Name = "series")

Reply:
Good to know